### ISSUE TITLE
[beA] Reihenfolge ändern führt zu Alias-Änderung

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#663]
![image](https://user-images.githubusercontent.com/29924198/74694313-f5bc1880-51ef-11ea-8368-42fa885c68ad.png)

Ich habe den Alias eines Dokumentes verändert. Dann schiebe ich ein anderes(!) Dokument in der Reihenfolge umher. Sofort wird wieder das Standard-Alias hergestellt und meine Änderung verworfen.

**Comment by stm9x9** [comment]
Label ist Question. Hast Du eine?

**Comment by j-dimension** [comment]
Es bleibt Question bis ich es als Bug verifiziert habe. 

Eine Frage an DICH wäre es, wenn es Dir assigned wäre :-)

**Comment by stm9x9** [comment]
der Fehler tritt leider immer noch auf

**Comment by j-dimension** [comment]
Kann ich nicht reproduzieren. Bitte kommentieren von welchem Wert auf welchen umbenannt wird, evtl. ist es wertabhängig. Danke!

**Comment by stm9x9** [comment]
Ich importiere ein paar Dateien
![image](https://user-images.githubusercontent.com/29924198/96299724-bf3fd300-0ff4-11eb-9ef9-82782dd1d26d.png)

bei der ersten ändere ich den alias auf test
![image](https://user-images.githubusercontent.com/29924198/96299764-d252a300-0ff4-11eb-843c-b5853ed9ff9c.png)

klicke auf eine beliebige andere Datei und dann auf Pfeil runter
![image](https://user-images.githubusercontent.com/29924198/96299792-de3e6500-0ff4-11eb-9c24-e050d2550329.png)

Die Datei ist verschoben und die ganz oben hat nicht mehr Test im alias
![image](https://user-images.githubusercontent.com/29924198/96299829-ec8c8100-0ff4-11eb-826f-fd27824e3569.png)

**Comment by Maz111111** [comment]
Das passiert bei diversen Sachen, die man nach der Umbenennung noch macht.

Zum Beispiel wenn man eine Datei von Anhang auf Schriftsatz ändert oder wenn man (von Hand) das AZ des Empfängers einträgt.

Da ich gerade einen Antrag an das Gericht mit vielen Anlagen, das vierte Mal neu benennen muss, finde ich aktuell, dass das ein ganz schön dringender Bug ist.

*Wegduck*
;-)

**Comment by j-dimension** [comment]
So dringend ist das m.E. nicht. Wenn Ihr erstmal die Kategorisierung nach Anlage oder Schriftsatz macht und dann die Aliasumbenennung, müsste es doch passen, oder?

**Comment by j-dimension** [comment]
Ich bin hier ehrlich gesagt im Unklaren ob es überhaupt irgendetwas zu tun gibt. Die Aliasumsetzung wurde gemacht, weil sich insbesondere die Gerichte im Dateinamen / Alias eine laufende Nr. und eine Angabe "Schriftsatz oder Anlage" wünschen. 

Ändert Ihr den Wert hinter diesen Angaben, also macht bspw. aus 

04_Anlage_irgendwas.odt

den Wert 

04_Anlage_Bild-blabla

so bleibt dieser auch erhalten, selbst wenn man die laufende Nr. durch Verschieben ändert oder von Anlage auf Schriftsatz wechselt. 

Macht man daraus - wie im Beispiel oben - "Test", dann erfüllt der Alias seine Funktion eigentlich nicht mehr.

**Comment by j-dimension** [comment]
https://www.gesetze-im-internet.de/ervv/__2.html

(2) Der Dateiname soll den Inhalt des elektronischen Dokuments schlagwortartig umschreiben und bei der Übermittlung mehrerer elektronischer Dokumente eine logische Nummerierung enthalten.

**Comment by Maz111111** [comment]
Ich nutze die Umbenennung, um die beigefügten Anlagen als K1 - Kirgendwas zu bezeichnen, da ich die elektronischen Dokumente teilweise wegen pdf/A nicht (digital) nachbearbeiten - "stempeln" - kann und auch nicht extra ausdrucken, stempeln und wieder einscannen wollte. Auch will ich die Namen der Dateien nicht K1 etc. ändern. Es handelt sich bei den Anlagen meist um Schriftsätze, die in der Akte gespeichert und entsprechend verschlagwortet sind.

Ich weiß aber auch gar nicht, ob das Gericht - welches ja derzeit noch ausdruckt - die gedruckten Anlagen entsprechend der Aliasnamen markiert. Vermutlich brauche ich dafür dann eine andere Lösung, was dann aber auch den Alias unnötig macht.

Lange Rede schwacher Sinn:

Es ist aber völlig richtig, dass man das Problem umgehen kann, wenn man das Verhalten der KSW kennt und beachtet.

**Comment by j-dimension** [comment]
> Ich nutze die Umbenennung, um die beigefügten Anlagen als K1 - Kirgendwas zu bezeichnen

Würde es etwas bringen, die laufende Nummer nicht mehr als 01...12 sondern K1...K12 zu generieren? Ist dieses K.. (für was steht das K?) irgendein informeller "Standard"?

**Comment by PBaumfalk** [comment]
K heißt „Kläger“
B heißt „Beklagter“
AS heißt „Antragsteller“
AG heißt „Antragsgegner“

Ist informeller Standard.

Mit freundlichen Grüßen / Kindly Regards

Patrick Baumfalk
Rechtsanwalt / Lawyer


Mobil:    +49 1525 53 04 194
E-Mail:   patrick.baumfalk@kanzlei-baumfalk.de


Baumfalk - Rechtsanwaltskanzlei
Hauptstr. 195
50169 Kerpen

Tel.: +49 2273 40 68 504
Fax: +49 2273 59 03 565

E-Mail: info@kanzlei-baumfalk.de
Web: www.kanzlei-baumfalk.de

**Comment by stm9x9** [comment]
> https://www.gesetze-im-internet.de/ervv/__2.html
> 
> (2) Der Dateiname soll den Inhalt des elektronischen Dokuments schlagwortartig umschreiben und bei der Übermittlung mehrerer elektronischer Dokumente eine logische Nummerierung enthalten.

Das betrifft den Dateinamen und gerade nicht den Alias bzw im ERV-Jargon Anzeigename. Meiner Ansicht nach ist gerade die logische Nummerierung für "Ausdrucker"-Gerichte gedacht, damit die Sachen in der richtigen Reihenfolge ausgedruckt werden.

Die Gericht die mit einer eAkte arbeiten interessieren sich nicht für den Dateinamen sondern den Anzeigenamen. Und da ist eine Reihenfolge völlig egal, da muß nur ein sinnhafter Name hin. siehe #757

**Comment by stm9x9** [comment]
> > Ich nutze die Umbenennung, um die beigefügten Anlagen als K1 - Kirgendwas zu bezeichnen
> 
> Würde es etwas bringen, die laufende Nummer nicht mehr als 01...12 sondern K1...K12 zu generieren? 
Nein, das hilft leider nicht. Denn wenn ich einen zweiten Schriftsatz hinterher schicke und weitere Anlagen ans Gericht schicke, dann würde j-l die in der beA-Nachricht wieder als K1, K2 ... benennen. Es ist dann aber zb K7, weil 1-6 mit der Klage schon verschickt wurden.

Daher denke ich 1, 2 ... sollte an den Dateinamen. Ebenso wie Anlage, Schriftsatz etc. Wobei hier auch immer noch das Problem mit der falschen Benennung als Anlage besteht. #664 

Der Anzeigenname hingegen sollte von jedem Automatismus frei sein, weil der Anwalt das selber benennen muß. Je nach Gusto nur als K1, K2 ... oder mit mehr Inhalt für das Gericht "K1 Scheidungsurkunde"

Das einzige, was man hier vorgeben könnte ist der Dateiname ohne Dateiendung in der Hoffnung, daß der Anwalt den schon in j-l gut sprechend benannt hat. Aber das auch nur einmal am Anfang beim Anhängen der Datei und nicht nach'm Verschieben oder so immer wieder. Dann kann der Anwalt passend zum Schriftsatz da noch ein K1 vorsetzen und fertig.
